frontend/config/application.coffee

```diff
@@ -8,8 +8,8 @@
 
 module.exports = require(process.env['LINEMAN_MAIN']).config.extend('application', {
   removeTasks:
-    common: [ "webfonts:dev", "images:dev"]
-    dist: ["images:dist", "webfonts:dist", "pages:dist"]
+    common: ['copy:dev', 'webfonts:dev', 'images:dev', 'concat_sourcemap', 'pages:dev']
```

Comment:
Does the `removeTasks` remove these from development mode, production mode or both?